### PR TITLE
Use correct sdk name in build_ios_app docs

### PR DESCRIPTION
**Pull request by robmoorman**
Leaving 10.0 or 11.1 causes the build task to fail

### CODE REVIEW

**Comment by robmoorman** [comment]
Rerun, passing cla's

File: fastlane/lib/fastlane/actions/build_ios_app.rb [hunk 1]
```diff
@@ -121,7 +121,7 @@ def self.example_code
             clean: true,
             output_directory: "path/to/dir", # Destination directory. Defaults to current directory.
             output_name: "my-app.ipa",       # specify the name of the .ipa file to generate (including file extension)
-            sdk: "10.0"                      # use SDK as the name or path of the base SDK when building the project.
+            sdk: "iOS 11.1"                  # use SDK as the name or path of the base SDK when building the project.
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/build_ios_app.rb]: Thanks for your contribution @robmoorman, and welcome to the fastlane community 🚀  Quick question: Does it work without the `iOS ` piece, or is this part required now? Can you try just `iOS 11.1` and see if that works also?

**Comment by robmoorman** [comment]
Hi @KrauseFx , leaving the `iOS` out gives the following error:

```
[21:52:09]: ▸ xcodebuild: error: SDK "11.1" cannot be located.
xcodebuild: error: SDK "11.1" cannot be located.
[21:52:09]: Exit status: 64
```

`xcodebuild -showsdks` gives:

```
iOS SDKs:
	iOS 11.1                      	-sdk iphoneos11.1

iOS Simulator SDKs:
	Simulator - iOS 11.1          	-sdk iphonesimulator11.1

macOS SDKs:
	macOS 10.13                   	-sdk macosx10.13

tvOS SDKs:
	tvOS 11.1                     	-sdk appletvos11.1

tvOS Simulator SDKs:
	Simulator - tvOS 11.1         	-sdk appletvsimulator11.1

watchOS SDKs:
	watchOS 4.1                   	-sdk watchos4.1

watchOS Simulator SDKs:
	Simulator - watchOS 4.1       	-sdk watchsimulator4.1
```

**Review by KrauseFx — APPROVED** [review]
Perfect, thanks for confirming @robmoorman 👍 This will be part of the next release

**Pull request merged**